### ISSUE TITLE
[French] Missing parameter ?

### ISSUE THREAD

**Issue by raiseandfall** [discogs/discogs-i18n#61]
Hi guys,

I merged #59 and fixed conflicts & missing translations on messages_js.po ( PR #60 ), and when reviewing the file I ran into a potential missing parameter :
```python
msgid "Rate this release  star."
msgid_plural "Rate this release  stars."
msgstr[0] "Évaluez cette référence  étoile."
msgstr[1] "Évaluez cette référence  étoiles."
...
...
msgid "This release is rated  star"
msgid_plural "This release is rated  stars"
msgstr[0] "Cet article est évalué à  étoile"
msgstr[1] "Cet article est évalué à  étoiles"
```

It looks like the number of stars is missing as a parameter.

**Comment by deleteme** [comment]
Yeah there is a flaw in this string. It should have `<%- n %>` in it. We'll fix this one soon. Thanks for pointing it out!

**Comment by deleteme** [comment]
This is fixed by #63.
